lib/resque/durable/guid.rb

```diff
@@ -1,24 +1,11 @@
+require 'uuidtools'
+
 module Resque
   module Durable
     module GUID
-
       def self.generate
-        [ hostname,
-          Process.pid,
-          Time.now.to_i,
-          increment_counter
-        ].join('/')
-      end
-
-      def self.hostname
-        @hostname ||= `hostname`.chomp
+        UUIDTools::UUID.random_create.to_s
```

Comment:
I'm not sure that we do, when this gets merged it can be versioned as a major if we want it to be, and then we can test the major version bump of the gem in staging.

If there are any issues in staging or in prod, we can revert to the old behavior by just reverting the version of the gem in the classic gemfile.

Reply:
Not sure we need to do this. We can test changes on staging before going to production.